packages/orbit-components/transforms/tokens-v16.mjs

```diff
@@ -80,63 +93,63 @@ const elevationTransformations = [
-  { from: /space(-[x|y])?-xxxl/, to: "space$1-[52px]" },
+  { from: /space(-[x|y])?-xxxs\b/, to: "space$1-50" },
+  { from: /space(-[x|y])?-xxs\b/, to: "space$1-100" },
+  { from: /space(-[x|y])?-xs\b/, to: "space$1-200" },
+  { from: /space(-[x|y])?-sm\b/, to: "space$1-300" },
+  { from: /space(-[x|y])?-md\b/, to: "space$1-400" },
+  { from: /space(-[x|y])?-lg\b/, to: "space$1-600" },
+  { from: /space(-[x|y])?-xl\b/, to: "space$1-800" },
+  { from: /space(-[x|y])?-xxl\b/, to: "space$1-1000" },
+  { from: /space(-[x|y])?-xxxl\b/, to: "space$1-[52px]" },
   // tailwind spacing, inset and translate classes
-  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xxxs/, to: "$1$2-50" },
-  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xxs/, to: "$1$2-100" },
-  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xs/, to: "$1$2-200" },
-  { from: /(p|m)([s|e|t|b|l|r|x|y])?-sm/, to: "$1$2-300" },
-  { from: /(p|m)([s|e|t|b|l|r|x|y])?-md/, to: "$1$2-400" },
-  { from: /(p|m)([s|e|t|b|l|r|x|y])?-lg/, to: "$1$2-600" },
-  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xl/, to: "$1$2-800" },
-  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xxl/, to: "$1$2-1000" },
-  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xxxl/, to: "$1$2-[52px]" },
-  { from: /(inset|gap|translate)(-[x|y])?-xxxs/, to: "$1$2-50" },
-  { from: /(inset|gap|translate)(-[x|y])?-xxs/, to: "$1$2-100" },
-  { from: /(inset|gap|translate)(-[x|y])?-xs/, to: "$1$2-200" },
-  { from: /(inset|gap|translate)(-[x|y])?-sm/, to: "$1$2-300" },
-  { from: /(inset|gap|translate)(-[x|y])?-md/, to: "$1$2-400" },
-  { from: /(inset|gap|translate)(-[x|y])?-lg/, to: "$1$2-600" },
-  { from: /(inset|gap|translate)(-[x|y])?-xl/, to: "$1$2-800" },
-  { from: /(inset|gap|translate)(-[x|y])?-xxl/, to: "$1$2-1000" },
-  { from: /(inset|gap|translate)(-[x|y])?-xxxl/, to: "$1$2-[52px]" },
-  { from: /(top|bottom|left|right|start|end])-xxxs/, to: "$1-50" },
-  { from: /(top|bottom|left|right|start|end)-xxs/, to: "$1-100" },
-  { from: /(top|bottom|left|right|start|end)-xs/, to: "$1-200" },
-  { from: /(top|bottom|left|right|start|end)-sm/, to: "$1-300" },
-  { from: /(top|bottom|left|right|start|end)-md/, to: "$1-400" },
-  { from: /(top|bottom|left|right|start|end)-lg/, to: "$1-600" },
-  { from: /(top|bottom|left|right|start|end)-xl/, to: "$1-800" },
-  { from: /(top|bottom|left|right|start|end)-xxl/, to: "$1-1000" },
-  { from: /(top|bottom|left|right|start|end)-xxxl/, to: "$1-[52px]" },
+  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xxxs\b/, to: "$1$2-50" },
+  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xxs\b/, to: "$1$2-100" },
+  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xs\b/, to: "$1$2-200" },
+  { from: /(p|m)([s|e|t|b|l|r|x|y])?-sm\b/, to: "$1$2-300" },
+  { from: /(p|m)([s|e|t|b|l|r|x|y])?-md\b/, to: "$1$2-400" },
+  { from: /(p|m)([s|e|t|b|l|r|x|y])?-lg\b/, to: "$1$2-600" },
+  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xl\b/, to: "$1$2-800" },
+  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xxl\b/, to: "$1$2-1000" },
+  { from: /(p|m)([s|e|t|b|l|r|x|y])?-xxxl\b/, to: "$1$2-[52px]" },
```

Comment:
Btw, while here, I got confirmation from design that `xxxl` should be replaced by `-1200` so please update the instances accordingly 🙏

Reply:
Done. ✅